scripts/update-apk.sh

```diff
@@ -18,12 +18,15 @@ cd apk
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
 	/bin/rm -f *
 else
-	/bin/rm -f app-debug.apk app-release.apk
+	/bin/rm -f app-debug.apk app-release.apk app-fdroid-debug.apk app-fdroid-release.apk
```

Comment:
This should be `app-playStore-debug.apk` right? same for release

Reply:
Yes